Description: Disable bundler and simplecov
 Disable the use of bundler as well as simplecov while running the tests
Author: Balasankar C <balasankarc@autistici.org>
Forwarded: not-needed
Last-Update: 2015-05-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -1,29 +1,5 @@
 require 'rubygems'
 
-using_git = File.exist?(File.expand_path('../../.git/', __FILE__))
-require 'bundler/setup' if using_git
-
-if RUBY_VERSION.to_f >= 1.9 && RUBY_ENGINE == 'ruby'
-  require 'simplecov'
-
-  SimpleCov.start do
-    add_filter "/spec"
-    add_filter "/features"
-    add_filter "/bin"
-    add_filter "/bundle"
-
-    # internet_connection mostly contains logic copied from the ruby 1.8.7
-    # stdlib for which I haven't written tests.
-    add_filter "internet_connection"
-  end
-
-  SimpleCov.at_exit do
-    File.open(File.join(SimpleCov.coverage_path, 'coverage_percent.txt'), 'w') do |f|
-      f.write SimpleCov.result.covered_percent
-    end
-    SimpleCov.result.format!
-  end
-end
 
 require "support/fixnum_extension"
 require "support/limited_uri"
